tests/code_size/hello_webgl2_wasm2js.json

```diff
@@ -1,10 +1,10 @@
 {
   "a.html": 588,
   "a.html.gz": 386,
-  "a.js": 21668,
-  "a.js.gz": 8361,
+  "a.js": 21688,
+  "a.js.gz": 8372,
   "a.mem": 3171,
   "a.mem.gz": 2715,
-  "total": 25435,
-  "total_gz": 11462
+  "total": 25427,
```

Comment:
How could the size of "a.js" have gone up but the "total" size gone down? Why is "total" undercounting the sizes of "a.html", "a.js", and "a.mem", which add up to 25447? Why did any sizes go up at all? Is that from unrelated changes that were hidden in slack?

Reply:
Good question.   Actually its looks like these codesize tests were not actually affected by this change.

I had run `test_minimal_runtime_code_size` with `REBASELINE_TESTS=1` on the assumption that they might change.   Because `test_minimal_runtime_code_size` as a slop factor the results tend to drift by a few bytes over time.. which means that if you run `REBASELINE_TESTS=1` at any given time something like this change will be created.

Its kind of sad because it makes it hard to know precisely what change is responsible for each shift, but its also makes this test a little less fragile.

I think this slop factor might be the explaination for the section question too.  I think pehaps there is bug where REBASELINE_TESTS should ignore slop but is not resulting in slightly strange results here... I'm investigating now. 

Anyway I reverted this part of the change so this is good to land now.